### PR TITLE
Possible Future<GVRMeshEyePointee> API

### PR DESCRIPTION
**Pull request by JonShemitz**
GearVRf-DCO-1.0-Signed-off-by: Jon Shemitz
j.shemitz@samsung.com

### CODE REVIEW

**Comment by rongguodong** [comment]
The code seems fine to me, and it works with Future<GVRMesh> correctly.

File: GVRf/Framework/src/org/gearvrf/GVRSceneObject.java [hunk 1]
```diff
@@ -431,6 +431,40 @@ public void attachEyePointeeHolder(GVREyePointeeHolder eyePointeeHolder) {
...
+            return false;
+        }
+
+        GVREyePointeeHolder eyePointeeHolder = new GVREyePointeeHolder(
+                getGVRContext());
+        eyePointeeHolder.addPointee(eyePointee);
+        attachEyePointeeHolder(eyePointeeHolder);
+        return true;
+    }
+
```
Comment (chenchao1407) [inline GVRf/Framework/src/org/gearvrf/GVRSceneObject.java]: Imho, I am wondering if the method name s/b attachDefaultEyePointeeHolder() which may make more sense.

**Comment by chenchao1407** [comment]
Tried using most of APIs you added here except getMeshEyePointee(Can't think out a use case right now). Works fine. Only a small concern on the method name, and also a little bit concern on those critical section( I may be over-thinking). But I think probably it should be good to go.

File: GVRf/Framework/src/org/gearvrf/GVRSceneObject.java (hunk 1 above)
Comment (JonShemitz) [inline GVRf/Framework/src/org/gearvrf/GVRSceneObject.java]: Perhaps. I think, though, that overloading the existing `attachEyePointeeHolder()` is a good idea: emphasizes the similarity.

**Pull request merged**